### PR TITLE
PIM July Serenity Update for the new associations type with quantities.

### PR DESCRIPTION
**Pull request by felix-couprie**
Ready for July 5th release.

### CODE REVIEW

File: content/updates/2020-07/1-associations-with-quantities.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+::: meta-data type="New" features="Product Associations" available="early July" link-to-doc="../articles/manage-your-association-types.html#create-an-association-type-with-quantities"
+:::
+
+With the new association type with quantities, all of our clients are now able to create a product set composed of other products that are associated with different quantities for each one.
+
+For example, it is now possible to create a dining table set composed of 1 table and 4 chairs, just like this one (picture courtesy of Made.com):
+
+![This is a product set composed of different associated products with quantities](../img/ryland-extending-dining-table-and-4-chairs-set-walnut-and-black_made.com.png)
+
+This new feature was highly requested, why not giving it a try right now ? :)
```
Comment (NolwennP) [inline content/updates/2020-07/1-associations-with-quantities.md:19]:
```suggestion
This new feature was highly requested, why not giving it a try right now? :)
```

File: content/updates/2020-07/1-associations-with-quantities.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# New: association type with quantities
+::: meta-data type="New" features="Product Associations" available="early July" link-to-doc="../articles/manage-your-association-types.html#create-an-association-type-with-quantities"
+:::
+
+With the new association type with quantities, all of our clients are now able to create a product set composed of other products that are associated with different quantities for each one.
```
Comment (NolwennP) [inline content/updates/2020-07/1-associations-with-quantities.md:13]: You should use "you" and not "all of our clients".

File: content/updates/2020-07/1-associations-with-quantities.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+# New: association type with quantities
+::: meta-data type="New" features="Product Associations" available="early July" link-to-doc="../articles/manage-your-association-types.html#create-an-association-type-with-quantities"
+:::
+
+With the new association type with quantities, you are now able to create a product set composed of other products that are associated with different quantities for each one.
+
+For example, it is now possible to create a dining table set composed of 1 table and 4 chairs, just like this one (picture courtesy of Made.com):
+
+![This is a product set composed of different associated products with quantities](../img/ryland-extending-dining-table-and-4-chairs-set-walnut-and-black_made.com.png)
```
Comment (LaureBro) [inline content/updates/2020-07/1-associations-with-quantities.md:17]: Hey Felix, could you also add the render of this pack in the PIM to make it more visual and impactful to users? 
Ie how this great Made.com product set is configured in Akeneo, so a screenshot with the table, the chairs and the quantities.
It's important to show how it's done in Akeneo. 
Thank you!

File: content/updates/2020-07/1-associations-with-quantities.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+This new feature was highly requested, why not giving it a try right now? ;)
```
Comment (LaureBro) [inline content/updates/2020-07/1-associations-with-quantities.md:19]:
```suggestion
This new feature was highly requested, why not giving it a try right now? :wink:
```

**Comment by camillebourjade** [comment]
Hi there, I believe the right way to go here is for @LaureBro to ping me when she collected and proofread all the final versions is that correct? Have a great day! bisous

**Comment by NolwennP** [comment]
Indeed, Camille won't review this PR. She'll review the English once it's merged.
The complete process for the Serenity updates is described in the "Creating Serenity updates content" Notion page. Please read it carefully, @felix-couprie

File: content/updates/2020-07/1-associations-with-quantities.md (hunk 3 above)
Comment (felix-couprie) [reply content/updates/2020-07/1-associations-with-quantities.md]: Hello Laure, I didn't have the possibility to display the image from Made.com dure to IP issue. So to meet your request I had to take pictures of my own furniture (less glamour) to create a real dining set with a table and 4 chairs. I've created them in the PIM also.
What do you think?

File: content/updates/2020-07/1-associations-with-quantities.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+For example, it is now possible to create a dining table set composed of 1 table and 4 chairs, just like this one:
+
+![This is a product set composed of different associated products with quantities](../img/Dining-set_Felix.jpg)
+
+In the PIM, it would look like this:
+
+![This is a product set composed of different associated products with quantities in the PIM](../img/Dining-set_Felix_PIM.png)
+
+This new feature was highly requested, why not giving it a try right now? :wink:
```
Comment (LaureBro) [inline content/updates/2020-07/1-associations-with-quantities.md:23]:
```suggestion
Association with quantities was one of the most requested features, so why not giving it a try right now? :wink:
```

File: content/updates/2020-07/1-associations-with-quantities.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+---
+pim_announcement_img: ./img/Dining-set_Felix.jpg
+pim_announcement_alt_img: "New association type with quantities"
+pim_announcement_audience:
+- CE
+- EE
+---
+
+# New: association type with quantities
```
[the remark this answers is not in the document]
Comment (felix-couprie) [reply content/updates/2020-07/1-associations-with-quantities.md:9]: OK

File: content/updates/2020-07/1-associations-with-quantities.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+::: meta-data type="New" features="Product Associations" available="early July" link-to-doc="../articles/manage-your-association-types.html#create-an-association-type-with-quantities"
+:::
+
+With the new association type with quantities, you are now able to create a product set composed of other products that are associated with different quantities for each one.
```
[the remark this answers is not in the document]
Comment (felix-couprie) [reply content/updates/2020-07/1-associations-with-quantities.md:13]: OK

File: content/updates/2020-07/1-associations-with-quantities.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
...
+
+For example, it is now possible to create a dining table set composed of 1 table and 4 chairs, just like this one:
```
[the remark this answers is not in the document]
Comment (felix-couprie) [reply content/updates/2020-07/1-associations-with-quantities.md:15]: OK

File: content/updates/2020-07/1-associations-with-quantities.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+
+![This is a product set composed of different associated products with quantities](../img/Dining-set_Felix.jpg)
+
+In the PIM, it would look like this:
```
[the remark this answers is not in the document]
Comment (felix-couprie) [reply content/updates/2020-07/1-associations-with-quantities.md:19]: OK

File: content/updates/2020-07/1-associations-with-quantities.md (hunk 5 above)
Comment (felix-couprie) [reply content/updates/2020-07/1-associations-with-quantities.md:23]: OK

File: Makefile [hunk 10]
```diff
@@ -36,8 +36,8 @@ test-firebase-functions: firebase-install
...
 
 deploy: build
-	$(DOCKER_RUN) -v /etc/passwd:/etc/passwd:ro -v $${SSH_AUTH_SOCK}:/ssh-auth.sock:ro -e SSH_AUTH_SOCK=/ssh-auth.sock $(DOCKER_IMAGE_TAG) rsync --no-v -e "ssh -q -p $${PORT} -o StrictHostKeyChecking=no" -az --delete dist/pim/serenity/ akeneo@$${HOSTNAME}:/home/akeneo/pim/serenity
-	$(DOCKER_RUN) -v /etc/passwd:/etc/passwd:ro -v $${SSH_AUTH_SOCK}:/ssh-auth.sock:ro -e SSH_AUTH_SOCK=/ssh-auth.sock $(DOCKER_IMAGE_TAG) rsync --no-v -e "ssh -q -p $${PORT} -o StrictHostKeyChecking=no" -az dist/pim/versions.json akeneo@$${HOSTNAME}:/home/akeneo/pim/versions.json
+	$(DOCKER_RUN) -v /etc/passwd:/etc/passwd:ro -v $${SSH_AUTH_SOCK}:/ssh-auth.sock:ro -e SSH_AUTH_SOCK=/ssh-auth.sock $(DOCKER_IMAGE_TAG) rsync --no-v -e "ssh -q -p $${PORT} -o StrictHostKeyChecking=no" -az --delete dist/pim/serenity/ akeneo@$${HOSTNAME}:/var/www/html/pim/serenity
+	$(DOCKER_RUN) -v /etc/passwd:/etc/passwd:ro -v $${SSH_AUTH_SOCK}:/ssh-auth.sock:ro -e SSH_AUTH_SOCK=/ssh-auth.sock $(DOCKER_IMAGE_TAG) rsync --no-v -e "ssh -q -p $${PORT} -o StrictHostKeyChecking=no" -az dist/pim/versions.json akeneo@$${HOSTNAME}:/var/www/html/pim/versions.json
```
Comment (NolwennP) [inline Makefile:40]: This file change has nothing to do in your PR. Do you know why it's here?

File: content/md/productivity/manage-your-rules.md [hunk 11]
```diff
@@ -100,10 +100,9 @@ Depending on the action you choose, the template changes. The table below lists
 | Clear associations  | Delete all the associations of your product/product models selection  |
```
Comment (NolwennP) [inline content/md/productivity/manage-your-rules.md:100]: Same here!

**Pull request merged**